mlrun/__main__.py

```diff
@@ -1022,6 +1032,9 @@ def project(
     timeout,
     ensure_project,
     schedule,
+    overwrite_schedule,
+    notification,
+    notifications,
```

Comment:
not sure need both params like this with click, the first one catches all the aliases

Reply:
Thanks, fixed 👍